files/php_custom.ini

```diff
@@ -1,6 +1,6 @@
 opcache.memory_consumption = 128
 opcache.interned_strings_buffer = 8
-opcache.max_accelerated_files = 4000
+opcache.max_accelerated_files = 30000
```

Comment:
@dpi I heard from Adam there might be some more things we should tweak. Any thoughts?